src/Features/Lsif/GeneratorTest/FoldingRangeTests.vb

```diff
@@ -54,7 +54,7 @@ using System.Linq;|}", "imports", "...")>
                     </Workspace>, openDocuments:=False, composition:=TestLsifOutput.TestComposition)
 
                 Dim annotatedLocations = Await AbstractLanguageServerProtocolTests.GetAnnotatedLocationsAsync(workspace, workspace.CurrentSolution)
-                Dim expectedRanges = annotatedLocations("foldingRange").Select(Function(location) CreateFoldingRange(rangeKind, location.Range, collapsedText)).OrderBy(Function(range) range.StartLine).ToArray()
+                Dim expectedRanges = annotatedLocations("foldingRange").Select(Function(location) CreateFoldingRange(rangeKind, location.Range, collapsedText)).OrderByDescending(Function(range) range.StartLine).ToArray()
```

Comment:
The order of the returned spans changed with the changes in this PR. I think the order shouldn't matter. So, I updated this test (and some others) to assert against the new order.

I think the tests could even ignore the order when comparing, but I didn't do that for now.

@CyrusNajmabadi Let me know your thoughts.